.circleci/config.yml

```diff
@@ -21,12 +21,20 @@ jobs:
             source /usr/local/share/virtualenvs/tap-facebook/bin/activate
             pylint tap_facebook -d C,R,W
       - run:
+          name: 'pylint tests'
+          command: |
+            source /usr/local/share/virtualenvs/tap-tester/bin/activate
+            pip install pylint
```

Comment:
Is there a reason we don't use `pylint` from the facebook venv?

Reply:
I needed to activate tap-tester so that we don't get import-error on all the tap-tester imports in the test. I assumed I needed to reinstall pylint, but maybe not?

Reply:
Oh right, you're running on the tests